tap_bing_ads/__init__.py

```diff
@@ -358,7 +357,7 @@ def discover_core_objects():
     return core_object_streams
 
 def get_report_schema(client, report_name):
-    column_obj_name = '{}Column'.format(report_name)
+    column_obj_name = '{}Column'.format(report_name) # pylint: disable=consider-using-f-string
```

Comment:
We can disable `consider-using-f-string` directly into the config.yml command and remove all the disabled comments in the code because it is present at multiple places.

Reply:
done